data_type/raster_formats.ipynb

```diff
@@ -0,0 +1,341 @@
+      "text/plain": [
+       "<Figure size 640x480 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "with rasterio.open(image_cog) as src:\n",
+    "    print(f\"Original image size: {src.width} x {src.height}\\n\")\n",
+    "\n",
+    "    # liste of overviews (for the first band)\n",
+    "    overview_levels = src.overviews(1)\n",
+    "\n",
+    "    # Iterate over the overviews\n",
+    "    for n_overview, overview_factor in enumerate(overview_levels, start=1):\n",
+    "        overview_width = src.width // overview_factor\n",
+    "        overview_height = src.height // overview_factor\n",
+    "        pixel_size = int(src.res[0] * overview_factor)\n",
+    "        print(f\"Overview {n_overview} of size {overview_width} x {overview_height} => pixel size: {pixel_size} meters\")\n",
+    "        start_time = time.time()\n",
+    "        overview_data = src.read(1, out_shape=(src.height // src.overviews(1)[n_overview - 1],  # read the data from the overview\n",
+    "                                               src.width // src.overviews(1)[n_overview - 1]))\n",
+    "        overview_time = time.time() - start_time\n",
+    "        print(f\"Load time {overview_factor}: {overview_time:.4f} seconds\\n\")\n",
+    "\n",
+    "    # plot the last overview to compare the quality of the image\n",
+    "    plt.imshow(overview_data, cmap=\"gray\")\n",
+    "    plt.title(f\"Overview using {pixel_size} meters pixels\")\n",
+    "    plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d2d62836-21c5-4622-a45e-9f3674cee0d6",
+   "metadata": {},
+   "source": [
+    "With only the second overview, the time needed to load the data into memory is divided by a factor of approximately 10 (this factor may vary depending on the machine you are using). The displayed image is the data loaded from the last overview (the one with the lowest resolution).\n",
+    "\n",
+    "## Conclusion\n",
```

Comment:
Thanks for this tutorial : it's simple and well explained. But maybe it would be interesting to leave user try with a bigger image, because with such size of images, it's hard to see differences.
Could you at least mention the read time and the size of a much bigger image (such as a full S2 product ?) ?

Reply:
Thank you for your feedback. I used this image because it could be automatically downloaded with a script. I couldn't find a full S2 image available for download without requiring an account on a provider’s website, which would have added complexity to the tutorial. Additionally, such an image would be too large to host directly in the repo.

I can add a subsection with read/write times and file sizes for all formats based on my tests I did with a real S2 image. I'll then include a short paragraph inviting users to download data from a provider and running the same benchmarks.

Reply:
We can probably find or add that, for example sharing the image on Zenodo.